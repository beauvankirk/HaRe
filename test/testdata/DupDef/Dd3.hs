module DupDef.Dd3 where

import DupDef.Dd1 hiding (dd)


f2 x = ff (x+1)

mm = 5
