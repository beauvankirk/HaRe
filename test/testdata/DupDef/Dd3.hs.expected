module DupDef.Dd3 where

import DupDef.Dd1 hiding (dd,mm)


f2 x = ff (x+1)

mm = 5
